library/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -58,7 +58,7 @@
     <TextView android:id="@id/exo_position"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:textSize="14sp"
+      android:textSize="13sp"
```

Comment:
Note also that you can override the layout in your own application code if you're programmatically replacing the font wish to do so. See [here](https://medium.com/google-exoplayer/customizing-exoplayers-ui-components-728cf55ee07a).